.circleci/common.py

```diff
@@ -34,9 +34,9 @@ def get_airflow_version(ac_version):
 
 
 IMAGE_MAP = collections.OrderedDict([
-    ("1.10.15-6", ["buster"]),
-    ("2.1.4-6", ["buster"]),
-    ("2.2.4-3", ["bullseye"]),
+    ("1.10.15-7.dev", ["buster"]),
+    ("2.1.4-7.dev", ["buster"]),
+    ("2.2.4-4.dev", ["bullseye"]),
```

Comment:
```suggestion
    ("1.10.15-7-dev", ["buster"]),
    ("2.1.4-7-dev", ["buster"]),
    ("2.2.4-4-dev", ["bullseye"]),
```

We need to switch to `-dev` instead of `.dev` so the generated version numbers are valid SemVer.

That should fix the tests.

(Apologies, I didn't announce this at all.)